src-built-in/components/appCatalog2/src/components/AppShowcase.jsx

```diff
@@ -0,0 +1,190 @@
+				images.push(this.props.app.images[i].url !== undefined ? this.props.app.images[i].url : "../assets/placeholder.svg");
+			} else {
+				images.push(this.props.app.images[newIndex].url !== undefined ? this.props.app.images[newIndex].url : "../assets/placeholder.svg");
+			}
+		}
+
+		return (
+			<div className="app-showcase">
+				<div className="header">
+					<img className="header-icon" src={iconUrl} />
+					<h3 className="appName">{name}</h3>
+					<button className="action-button">
+						{installed ? (
+							<span className="action-button-label">Open</span>
+						) : (
+							<span className="action-button-label">
+								<i className="ff-plus"></i>
+								"My Apps"
+							</span>
+						)}
+					</button>
+				</div>
+				<div className="image-carousel-container">
+					<div className="paginate_carat_left" onClick={this.previousImage} />
+					<div className="image-carousel">
+						{images.map((imageUrl) => {
+							return (
+								<img className='image-carousel-image' src={imageUrl} />
+							);
+						})}
+					</div>
+					<div className="paginate_carat_right" onClick={this.nextImage} />
+				</div>
+				<div className="app-notes description">
+					<span className="showcase-label">Description</span>
+					<div className="description-content">
+						<p>{this.props.app.description}</p>
+					</div>
+				</div>
+				<div className="app-notes release-notes">
+					<span className="showcase-label">Release Notes</span>
+					<div className="release-notes-content">
+						<p>{this.props.app.releaseNotes}</p>
+					</div>
+				</div>
+				<div className="app-notes developer">
+					<span className="showcase-label">Developer</span>
+					<div className="developer-content">
+						<p>{this.props.app.publisher} - <a href="#" onClick={this.openSite} >{this.props.app.contactEmail}</a></p>
+					</div>
+				</div>
+				<div className="dev-notes version-update">
+					<div className="version-content">
+						<span className="showcase-label">Version</span>
+						<div className="version">
+							{this.props.app.version}
+						</div>
+					</div>
+					<div className="updated-content">
+						<span className="showcase-label">Last Updated</span>
+						<div className="updated">
+							Not available
+						</div>
+					</div>
+				</div>
+				<div className="dev-notes support">
+					<div className="support-content">
+						<span className="showcase-label">Support</span>
+						<div className="support">
+							{this.props.app.supportEmail}
+						</div>
+					</div>
+					<div className="tags-content">
+						<span className="showcase-label">Tags</span>
+						<div className="tags">
+							{this.props.app.tags.map((tag) => {
+								let tagName = tag[0].toUpperCase() + tag.substring(1);
+
+								return (
+									<div className="tag-label">
+										<span className="label-content">{tagName}</span>
+									</div>
+								);
+							})}
+						</div>
+					</div>
+				</div>
+			</div>
+		);
```

Comment:
Lots of rendering in here :) Might be a good idea to decouple and break it into little components